src/ugeneui/src/main_window/ProxyStyle.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ProxyStyle.h"
+
+#include <QMenu>
+
+namespace U2 {
+
+ProxyStyle::ProxyStyle(QStyle *style)
+    : QProxyStyle(style)
+{
+
+}
+
+void ProxyStyle::polish(QWidget *widget) {
+    QProxyStyle::polish(widget);
+
+#if QT_VERSION_CHECK(5, 10, 0) < QT_VERSION
+    // It's a workaround for https://bugreports.qt.io/browse/QTBUG-49435:
+    // shortcuts are not shown in context menus.
+    auto *menu = qobject_cast<QMenu *>(widget);
+    if (nullptr != menu) {
```

Comment:
no problem: you do not need to use Yoda style with modern compilers. UGENE build will break if you have assignment within expressions in your code.

or read the text above like this (mimics the way devs reads the Yoda expressions in code):

Modern compilers with no Yoda style to use. Break will UGENE it once use anyone for bad.

Reply:
We use rather old compilers.